test/test.js

```diff
@@ -33,6 +33,13 @@ describe('chai-immutable', function () {
     });
 
     describe('equal method', function () {
+      it(
+        'fails when only the "expected" value is an Immutable collection',
+        function () {
+          assert.throws(expect([]).to.equal.bind(null, List()));
```

Comment:
Oh, sorry you're right. I was seeing the other failure. I just added this one because it seemed like there should be a test for both interfaces.